WordPress/Classes/ViewRelated/Jetpack/Branding/Button/JetpackButton.swift

```diff
@@ -74,7 +74,7 @@ class JetpackButton: UIButton {
         tintColor = buttonTintColor
         backgroundColor = buttonBackgroundColor
         setTitleColor(buttonTitleColor, for: .normal)
-        titleLabel?.font = Appearance.titleFont
+        titleLabel?.font = UIFontMetrics.default.scaledFont(for: Appearance.titleFont, maximumPointSize: 22)
```

Comment:
nit: can we put this whole thing inside Appearance.titleFont?

Reply:
Good call - I'd missed that it was local to the button and not the app. 👍

Reply:
This should be resolved with 72434a5da17fd28bd979ac2141ac14b8af2a1e3b. Thanks!